layers/meta-resin-imx6ul-var-dart/recipes-core/dropbear/dropbear_%.bbappend

```diff
@@ -0,0 +1 @@
+SECURITY_CFLAGS = "-fstack-protector-strong -pie -fpie"
```

Comment:
You can also add a newline here

Reply:
Maybe link to the exact security_flags.inc file file in the git log so we know what to reference (you can link to the file with the branch included in the link)